examples/website/3d-tiles/examples.js

```diff
@@ -2,7 +2,7 @@ const DECK_DATA_URI = 'https://raw.githubusercontent.com/uber-common/deck.gl-dat
 const ION_TOKEN =
   'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiJlYWMxMzcyYy0zZjJkLTQwODctODNlNi01MDRkZmMzMjIxOWIiLCJpZCI6OTYyMCwic2NvcGVzIjpbImFzbCIsImFzciIsImdjIl0sImlhdCI6MTU2Mjg2NjI3M30.1FNiClUyk00YH_nWfSGpiQAjR5V2OvREDq1PJ5QMjWQ';
 
-const DATA_URI = 'https://raw.githubusercontent.com/uber-web/loaders.gl/master';
+const DATA_URI = 'https://raw.githubusercontent.com/visgl/loaders.gl/master';
```

Comment:
Good catch. I just did a searh and notices that there is a lot of references to uber-web still in the code base. Maybe you could open a separate PR that replaces all of these?